extensions/kind/src/create-cluster.ts

```diff
@@ -46,24 +45,10 @@ export function getKindClusterConfig(
   });
 }
 
-function getTags(tags: Tags): Tags {
-  for (const tag of tags) {
-    if (typeof tag === 'object' && 'tag' in tag) {
-      if (tag.tag === 'tag:yaml.org,2002:int') {
-        const newTag = { ...tag };
-        newTag.test = /^(0[0-7][0-7][0-7])$/;
-        newTag.resolve = (str: string): number => parseInt(str, 8);
-        tags.unshift(newTag);
-        break;
-      }
-    }
-  }
-  return tags;
-}
-
 export async function setupIngressController(clusterName: string): Promise<void> {
-  const manifests = loadAllYaml(ingressManifests, { customTags: getTags });
-  await extensionApi.kubernetes.createResources('kind-' + clusterName, manifests);
+  const parsedManifest = parseAllDocuments(ingressManifests);
```

Comment:
let's make the conversion in a separate function (convertYamlFrom11to12(string): string), so we can just remove the call to the function when the issue is fixed (and the intent will be clearer)